src/instrumentsscene/view/instrumentsettingsmodel.cpp

```diff
@@ -61,46 +63,50 @@ void InstrumentSettingsModel::load(const QVariant& instrument)
 
 void InstrumentSettingsModel::replaceInstrument()
 {
-    if (!masterNotationParts()) {
-        return;
-    }
+    mu::async::Async::call(this, [this]() {
```

Comment:
Isn't there a chance that this breaks too, because if the popup content is deleted, the model is deleted too?

(Assuming that the problem was that the content of the StyledPopupView is deleted and thus the `callLater` doesn't happen)